mainmatter/introduction/graphics_hardware/index.tex

```diff
@@ -123,7 +123,45 @@ \subsubsection{NVIDIA}
 We were not able to find a whitepaper or similar report from reliable sources which detailed the GP106 chip, so that is why it is not used as the example. 
 
 
-\subsubsection{Intel}
+\subsubsection{AMD}
+For this project we have a AMD Sapphire Radeon R9 280 3GB GDDR5 \gls{GPU} at our disposal. It uses the GCN 1.0 Architecture.
+GCN stands for Graphics Core Next, it is the first architecture AMD developed that would allow to use \glspl{GPU} as \glspl{GPGPU}.
 
+\fig{figures/GCN_compute_unit}{Hello World}{GCNCU}{1} 
```

Comment:
"Hello World" er ikke særlig sigende. Hvad er det for en GPU vi ser på?

Reply:
fixed